README.md

```diff
@@ -47,7 +47,7 @@ This is a tool meant for VIM users. In other words, it is not a VIM plugin
 
 The only hard dependency is [SyntaxRange](https://github.com/vim-scripts/SyntaxRange).
 
-Other than that you may need some syntax highlighting plugins to handle the code syntax highlighting.
+Other than that you may need some syntax highlighting plugins to handle the code syntax highlighting. [vim-scripts/ingo-library](https://github.com/vim-scripts/ingo-library) may be required for syntax highlighting.
```

Comment:
There are many other plugins that do Markdown syntax highlighting. https://github.com/tpope/vim-markdown & https://github.com/plasticboy/vim-markdown are just two other examples. I don't know much about ingo-library, but I do know that recommending finding *any* markdown syntax highlighting plugin is safer than recommending a specific one. I'd be happy to merge this in if this part were taken out.